src/features/layer_features_panel/readme.md

```
@@ -0,0 +1,73 @@
+## Layer Features Panel
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)